lib/pengine/tests/status/set_working_set_defaults_test.c

```diff
@@ -28,7 +28,7 @@ check_defaults(void **state) {
             |pcmk_sched_stop_removed_resources
             |pcmk_sched_cancel_removed_actions;
 
-    if (!strcmp(PCMK__CONCURRENT_FENCING_DEFAULT, "true")) {
+    if (!strcmp(PCMK__CONCURRENT_FENCING_DEFAULT, PCMK_VALUE_TRUE)) {
```

Comment:
Okay in this context?

Reply:
Yes, because we're not comparing user input, we're comparing our own settings (via configure in this case)